### PR TITLE
Enable accuracy op for ngraph engine

### PR DESCRIPTION
**Pull request by baojun-nervana**

### CODE REVIEW

File: paddle/fluid/operators/ngraph/ops/accuracy_op.h [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+#include <string>
+#include <vector>
+#include "ngraph/ngraph.hpp"
+#include "paddle/fluid/platform/ngraph_helper.h"
+
+namespace paddle {
+namespace operators {
+namespace ngraphs {
+
+void BuildAccuracyNode(
```
Comment (tensor-tang) [inline paddle/fluid/operators/ngraph/ops/accuracy_op.h:26]: I saw this function seems common, could you extract it to helper?

Comment (baojun-nervana) [reply paddle/fluid/operators/ngraph/ops/accuracy_op.h:26]: This function is to map paddle accuracy operator to ngraph accuracy op, not really a common one.  Did I understand anything wrong here?

Comment (tensor-tang) [reply paddle/fluid/operators/ngraph/ops/accuracy_op.h:26]: It seems each new op would have function BuildXXXNode, I thought we could extract any functions to reuse code.

File: paddle/fluid/operators/ngraph/ops/top_k_op.h [hunk 2]
```diff
@@ -36,11 +36,6 @@ void BuildTopKNode(
       std::make_shared<ngraph::op::GetOutputElement>(top_k, 0);
   std::shared_ptr<ngraph::Node> out =
       std::make_shared<ngraph::op::GetOutputElement>(top_k, 1);
-  auto dummy_out = paddle::platform::GetOutputNode(op, "Out", ngb_node_map);
```
Comment (tensor-tang) [inline paddle/fluid/operators/ngraph/ops/top_k_op.h:39]: why delete this?

**Comment by tensor-tang** [comment]
Please solve the conflicts

File: paddle/fluid/operators/ngraph/ops/top_k_op.h (hunk 2 above)
Comment (baojun-nervana) [reply paddle/fluid/operators/ngraph/ops/top_k_op.h:39]: the element_type check was absorbed by SetOutputNode function. This became a duplicate

**Review by baojun-nervana — COMMENTED** [review]
conflict resolved

**Review by tensor-tang — APPROVED** [review]
LGTM

**Pull request merged**